index.js

```diff
@@ -0,0 +1,175 @@
+let express = require('express');
```

Comment:
use const if you don't want to change the var. Might be worth adding eslint config (but also might be overkill). We should at least run the code through the linter to make sure it's clean.

Reply:
Good point, I'll add it to the dependency list and add a script, but won't mention much of it in the tutorial.